UnicornEngine/video/include/unicorn/video/Camera.hpp

```diff
@@ -19,105 +19,100 @@ namespace unicorn
 namespace video
 {
 /**
- * @brief Type of projection, which we can use in Camera
+ * @brief Type of camera projection
  */
 enum class ProjectionType
 {
     Perspective
 };
 
 /**
- * @brief Camera class, which create View and Projection matrices.
+ * @brief Camera class, hold view and projection matrices.
  */
 class Camera
 {
 public:
     /**
      * @brief Camera constuctor
-     * @param postion Default position of camera
+     * @param position Default position of camera
```

Comment:
Remove all `of camera` and `for camera` from all descriptions since this class doesn't do anything that is **not** about the camera.